### ISSUE TITLE
Create protocol to dump a resource

### ISSUE THREAD

**Issue by darozak** [Darwin-River/Ex-Machinis#219]
I need to create a protocol for dumping unwanted resources into space.  This will be important not only for game-play but for testing game mechanics.

**Comment by darozak** [comment]
I programmed a dump protocol (Id = 1) into the latest version of the economy (https://github.com/Darwin-River/Ex-Machinis/commit/8ff5d2e325c97fbae038dbe8eb5621dd5188d5f3) and tested it online.  For some reason when I tried to dump iron pellets (Id = 2) from drone 52 using "2 1 perform" it ended up reporting that I had five pellets in email when I sent the "52 vPrintCargoList" command event though it correctly reported zero pellets in https://exmachinis.com/spacecraft/52, having reduced the number of iron pellets from 1 to 0 with the dump protocol.

Will need to look into the reason why vPrintCargo list shows five pellets and open a new issue if necessary.
